fabtests/include/shared.h

```diff
@@ -520,14 +525,43 @@ static inline uint64_t ft_gettime_ms(void)
 	return ft_gettime_ns() / 1000000;
 }
 
-static inline void ft_start(void)
+static inline void ft_timer_start(union ft_timer *timer)
```

Comment:
Is there a specific reason you changed this to pass in a timer instead of using a global? I'm not necessarily against it if you have a reason you want to update it like if you have a test you want to add that has multiple timers. It's just not necessary right now and makes for a lot of code changes without a clear reason.